baggage/templates/include/image.html

```diff
@@ -0,0 +1,3 @@
+{% if value %}
+    <img src="{{value.url}}" width="100%" />
```

Comment:
Could you add it on the same folder as the field one?

Reply:
I guess so, but if I remove the field then...

Reply:
I mean in the `app` one with the fields.html

Reply:
Sure then, but I wanted to keep this really separated from app.

Reply:
It can be useful to reuse in other apps (new or old)